crane/views/v1.py

```diff
@@ -48,6 +48,23 @@ def ping():
     return response
 
 
+@section.route('/repositories')
```

Comment:
We should keep this view outside the path space of the existing docker API. That will make it easy to restrict access to this view, make sure it doesn't get confused as being part of the docker API, avoid problems if the docker API ever tries to make use of this path space, etc. This is basically a non-public view for use by admins that should be independent of other data representations.